lib/IRGen/StructLayout.h

```diff
@@ -181,9 +171,7 @@ class ElementLayout {
     Index = nonFixedElementIndex;
   }
 
-  const TypeInfo &getTypeForLayout() const { return *TypeLayout; }
-
-  const TypeInfo &getTypeForAccess() const { return *TypeAccess; }
```

Comment:
This commit seems like a mechanical transformation, which is great, but does LLDB still need the static layout for now?

Reply:
LLDB globally disables resilience with a frontend flag, so its even less resilient than this hack was. :)